README.md

```diff
@@ -136,3 +143,308 @@ There are many ways to contribute:
+* [ifname](/plugins/processors/ifname)
+* [filepath](/plugins/processors/filepath)
+* [override](/plugins/processors/override)
+* [parser](/plugins/processors/parser)
+* [pivot](/plugins/processors/pivot)
+* [port_name](/plugins/processors/port_name)
+* [printer](/plugins/processors/printer)
+* [regex](/plugins/processors/regex)
+* [rename](/plugins/processors/rename)
+* [reverse_dns](/plugins/processors/reverse_dns)
+* [s2geo](/plugins/processors/s2geo)
+* [starlark](/plugins/processors/starlark)
+* [strings](/plugins/processors/strings)
+* [tag_limit](/plugins/processors/tag_limit)
+* [template](/plugins/processors/template)
+* [topk](/plugins/processors/topk)
+* [unpivot](/plugins/processors/unpivot)
+
+## Aggregator Plugins
+
+* [basicstats](./plugins/aggregators/basicstats)
+* [final](./plugins/aggregators/final)
+* [histogram](./plugins/aggregators/histogram)
+* [merge](./plugins/aggregators/merge)
+* [minmax](./plugins/aggregators/minmax)
+* [valuecounter](./plugins/aggregators/valuecounter)
+
+## Output Plugins
+
+* [influxdb](./plugins/outputs/influxdb) (InfluxDB 1.x)
+* [influxdb_v2](./plugins/outputs/influxdb_v2) ([InfluxDB 2.x](https://github.com/influxdata/influxdb))
+* [amon](./plugins/outputs/amon)
+* [amqp](./plugins/outputs/amqp) (rabbitmq)
+* [application_insights](./plugins/outputs/application_insights)
+* [aws kinesis](./plugins/outputs/kinesis)
+* [aws cloudwatch](./plugins/outputs/cloudwatch)
+* [azure_monitor](./plugins/outputs/azure_monitor)
+* [cloud_pubsub](./plugins/outputs/cloud_pubsub) Google Cloud Pub/Sub
+* [cratedb](./plugins/outputs/cratedb)
+* [datadog](./plugins/outputs/datadog)
+* [discard](./plugins/outputs/discard)
+* [dynatrace](./plugins/outputs/dynatrace)
+* [elasticsearch](./plugins/outputs/elasticsearch)
+* [exec](./plugins/outputs/exec)
+* [execd](./plugins/outputs/execd)
+* [file](./plugins/outputs/file)
+* [graphite](./plugins/outputs/graphite)
+* [graylog](./plugins/outputs/graylog)
+* [health](./plugins/outputs/health)
+* [http](./plugins/outputs/http)
+* [instrumental](./plugins/outputs/instrumental)
+* [kafka](./plugins/outputs/kafka)
+* [librato](./plugins/outputs/librato)
+* [logz.io](./plugins/outputs/logzio)
+* [mqtt](./plugins/outputs/mqtt)
+* [nats](./plugins/outputs/nats)
+* [newrelic](./plugins/outputs/newrelic)
+* [nsq](./plugins/outputs/nsq)
+* [opentsdb](./plugins/outputs/opentsdb)
+* [prometheus](./plugins/outputs/prometheus_client)
+* [riemann](./plugins/outputs/riemann)
+* [riemann_legacy](./plugins/outputs/riemann_legacy)
+* [socket_writer](./plugins/outputs/socket_writer)
+* [stackdriver](./plugins/outputs/stackdriver) (Google Cloud Monitoring)
+* [syslog](./plugins/outputs/syslog)
+* [tcp](./plugins/outputs/socket_writer)
+* [udp](./plugins/outputs/socket_writer)
+* [warp10](./plugins/outputs/warp10)
+* [wavefront](./plugins/outputs/wavefront)
+* [sumologic](./plugins/outputs/sumologic)
+* [yandex_cloud_monitoring](./plugins/outputs/yandex_cloud_monitoring)
```

Comment:
I don't think this is supposed to be included in your PR

Reply:
Thanks, will fix it